GUI/Forms/ExtractProgressForm.cs

```diff
@@ -26,7 +26,11 @@ public partial class ExtractProgressForm : Form
 
         private static readonly List<ResourceType> ExtractOrder = new()
         {
-            // Materials before textures
+            ResourceType.World,
+            ResourceType.WorldNode,
+            ResourceType.Model,
+            ResourceType.Mesh,
```

Comment:
I can Morph can be added to the list now, as master now parses it (and thus loads atlas texture).